Makefile

```diff
@@ -16,7 +16,7 @@ endif
 
 
 ## Functions
-get_github_latest_version = $$(docker run --rm easy_infra:latest /bin/bash -c "curl -s https://api.github.com/repos/$(1)/releases/latest | jq -r '.tag_name'")
+get_github_latest_version = $$(docker run --rm alpine:3.13 /bin/ash -c "apk add curl jq &>/dev/null && curl -s https://api.github.com/repos/$(1)/releases/latest | jq -r '.tag_name'")
```

Comment:
On line 32 Had to remove update-github to test make update locally.

Reply:
That shouldn't be an issue, can you do a `docker pull alpine:3.13 && make update-github` and see if that works?